tests/TestDeepScan.py

```diff
@@ -133,6 +133,29 @@ def test_delete(self):
         # clean up
         shutil.rmtree(base)
 
+    def test_normalized_walk_darwin(self):
+        import mock
+
+        with mock.patch('os.walk') as mock_walk:
```

Comment:
I like it. I should use the mock module in some other tests where I mocked up methods without it.